SPEC.md

```diff
@@ -64,50 +64,55 @@ It includes a *rootfs* with all of the files that will exist in the root of the
 
 ### Image Archives
 
-The ACI archive format aims for flexibility and relies on very boring technologies: HTTP, gpg, tar and gzip.
-This set of formats makes it easy to build, host and secure a container using technologies that are battle-tested.
+The ACI file format ("image archive") aims for flexibility and relies on standard and common technologies: HTTP, gpg, tar and gzip.
+This set of formats makes it easy to build, host and secure a container using technologies that are widely available and battle-tested.
 
-Image archives MUST be a tar formatted file.
-The image may be optionally compressed with gzip, bzip2, or xz.
-After compression, images may also be encrypted with AES symmetric encryption.
+- Image archives MUST be a tar formatted file.
+- All files in the image MUST maintain all of their original properties, including timestamps, Unix modes, and extended attributes (xattrs).
+- Image archives MUST be named with the suffix `.aci`, irrespective of compression/encryption (see below).
+- Image archives SHOULD be signed using PGP, in detached signature mode.
```

Comment:
After thinking through this more I think we should do late signing of the image. The downside is that we will need to store the type of compression used if we wish to check the signature but it is the right thing to do from a security perspective. In the next revision of the spec we can use something like JOSE so that the signature includes the type of compression used.